exporter/awsemfexporter/metric_translator.go

```diff
@@ -118,13 +124,8 @@ func (dps DoubleHistogramDataPointSlice) At(i int) DataPoint {
 	return dps.DoubleHistogramDataPointSlice.At(i)
 }
 
-// TranslateOtToCWMetric converts OT metrics to CloudWatch Metric format
-func TranslateOtToCWMetric(rm *pdata.ResourceMetrics, config *Config) ([]*CWMetrics, int) {
-	var cwMetricList []*CWMetrics
-	totalDroppedMetrics := 0
-	namespace := config.Namespace
-	var instrumentationLibName string
-
+func GetNamespaceFromMetric(rm *pdata.ResourceMetrics) (string) {
+	var namespace string
```

Comment:
What's the namespace for nginx workload? 
We need allow configurable namespace.

Reply:
This is just used to validate namespace check for empty or nil string